dubbo-spring-boot/dubbo-spring-boot-3-autoconfigure/pom.xml

```diff
@@ -68,9 +68,9 @@
     </dependency>
 
     <dependency>
-      <groupId>jakarta.servlet</groupId>
-      <artifactId>jakarta.servlet-api</artifactId>
-      <optional>true</optional>
+      <groupId>org.apache.tomcat.embed</groupId>
+      <artifactId>tomcat-embed-core</artifactId>
+      <scope>provided</scope>
```

Comment:
This change will cause users to have to use Tomcat.

Reply:
> This change will cause users to have to use Tomcat.
The scope is provided, which doesn't have any actual impact, just adds special support for Tomcat, as Tomcat already includes the servlet-api, so it can be replaced directly.